docs/munin.md

```diff
@@ -1,8 +1,10 @@
 # nf-core/configs: MUNIN Configuration
 
-All nf-core pipelines have been successfully configured for use on the MUNIN cluster aka big iron.
+All nf-core pipelines have been successfully configured for use on the MUNIN cluster.
 
-To use, run the pipeline with `-profile munin`. This will download and launch the [`munin.config`](../conf/munin.config) which has been pre-configured with a setup suitable for the MUNIN cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+To use, run the pipeline with `-profile munin`.
+This will download and launch the [`munin.config`](../conf/munin.config) which has been pre-configured with a setup suitable for the MUNIN cluster.
+Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
```

Comment:
You can run with either `docker` or `singularity` with this profile? May need to be clarified here.

Reply:
Yes, we can, but good point, I'll clarify that

Reply:
Made some clarification